component-models.json

```diff
@@ -2651,5 +2651,37 @@
         "description": "Label for the default region filter option. Will serve as an 'all' filter option."
       }
     ]
+  },
+  {
+    "id": "cookie-banner",
+    "fields": [
+      {
+        "component": "text-input",
```

Comment:
@iSupercell We don't need this I guess. The title and text can be authored together in RTE.